### ISSUE TITLE
Get value of time (t) in ode function

### ISSUE THREAD

**Issue by Dragomordor** [aseyboldt/sunode#55]
Hi, I have a term in my ode that is dependant on the exact time of evaluation. This is because I am look at a reactor with inflow data, but essentially it can be shown using an example like this:
```
ode(t, y, p, inflowdata):

   time_data = inflowdata[:, 0] # First column is time points for measured data

   # Get Qin at time t
   Qin = np.interp(t, time_data, inflowdata[:, 1] # Second column are Flowrates in at each time point
   
  ....
```

However, because it is symbolic I cannot just use it like that and I have tried a few solutions with no luck. Can anyone help me?

**Comment by Dimlitter** [comment]
Use `sympy`. t is a symbol type value. I met the same issue just few weeks ago lol

**Comment by Dragomordor** [comment]
So just use sympy.t?

**Comment by Dimlitter** [comment]
Obviously not. I don't konw exactly what you want to do, but I will give you an example of mine to show how to calculate it. 
Just like this:
![image](https://github.com/user-attachments/assets/214f221f-2eb5-49b9-a66a-eee8d9d77d32)

**Comment by Dragomordor** [comment]
Yeah from what I can tell you need a continuous function in terms of t, and I tested with piecewise functions (as well as symbolic interpolation) with no luck
